bedrock/settings/base.py

```diff
@@ -1177,7 +1177,50 @@ def lazy_email_backend():
 
 # country code for /country-code.json to return in dev mode
 DEV_GEO_COUNTRY_CODE = config('DEV_GEO_COUNTRY_CODE', default='US')
-SEND_TO_DEVICE_COUNTRIES = config('SEND_TO_DEVICE_COUNTRIES', default='US', cast=Csv())
+SEND_TO_DEVICE_MESSAGE_SETS = {
+    'default': {
+        'sms_countries': config('STD_COUNTRIES_DEFAULT', default='US', cast=Csv()),
```

Comment:
Should we have a public place for referencing which countries support SMS for each message set?

In addition, would be good to update the portion of [the docs](https://github.com/mozilla/bedrock/blob/master/docs/send-to-device.rst) referring to SMS (and GeoDude?):

>The form allows sending via SMS or Email, although the SMS copy & messaging is shown only to those in the US. Geo-location is handled in JavaScript using GeoDude.

Reply:
Great catch. I'll update.

Reply:
As for public, I was just going to rely on the published configs that are part of the configurator. Hopefully that's enough?